docs/getting-started/configure.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default 
+title: Configure GrimoireLab 
+nav_order: 2 parent: Getting-started
+has_children: true 
+has_toc: false
+---
+
+# Configure GrimoireLab
+
+GrimoireLab can be setup through the use of two main configuration files,
+`projects.json` and `setup.cfg`. The role and usage of each one of those files
+are described in their corresponding sections.
+
+- [projects.json](https://vsevagen.github.io/grimoirelab-tutorial/docs/getting-started/projects/)
```

Comment:
I think the link should be replaced

Reply:
I left it blank at the moment. I'll update it once we have the whole thing deployed.